notebooks/compendium/01.chsh/index.ipynb

```diff
@@ -13,7 +13,7 @@
    "id": "a0bed888",
```

Comment:
<blockquote><strong>Alice and Bob know their measurements will be the same, regardless of the basis they choose to measure in.</strong></blockquote><blockquote>By shifting the measurement basis slightly, it causes the probabilities to be shifted in the favor of Alice and Bob.</blockquote><p>I think a reader could find these two sentences confusing. Maybe something along these lines:</p><p> </p><p><em>"If Alice and Bob both measure using the same basis, their bits will always agree. But if they measure in slightly different bases, there's a chance their bits will disagree. Alice and Bob's tactic is to shift their measurement angles in opposite directions when the referee gives them a 1. This means they have a higher chance of their answers disagreeing when the referee gives them both a 1."</em></p><p> </p><p> </p><p>In terms of writing (reviewnb only allows one comment per cell):</p><ul><li>I think the writing style leans more towards a "journey", whereas the compendium should be more of a reference (this is something to discuss as a group).</li><li>I would add subheadings for classical and quantum strategies, and have the writing stick to the titles more strictly. For example, the "strategies" section continues describing the game.</li></ul><p> </p><p>Also, these facts are repeated throughout the section which convolutes the explanation a bit. For example:</p><ul><li>Using entanglement increases the winning probability</li><li>Alice and Bob are unable to communicate during the game</li><li>Alice and Bob decide a strategy before the game</li></ul><p>I think stricter section headings would mostly fix this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/textbook/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>